src/cdk/testing/kagekiri.d.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+declare module 'kagekiri' {
+  export function querySelectorAll(selector: string, root: Element);
```

Comment:
Not also `querySelector`?

Reply:
Nope, all the harness stuff runs off just qsa